packages/matchbox/src/components/Slider/Slider.js

```diff
@@ -44,6 +45,22 @@ function Slider(props) {
     }
   }, [sliderValue, windowSize]);
 
+  // Updates tick locations when ticks or window size change
+  React.useLayoutEffect(() => {
+    if (typeof ticks === 'object') {
```

Comment:
I think it is safe to tighten the prop types for `ticks` to `PropTypes.object` and loosen this check to existence (`if (ticks)`).

Reply:
woops `PropTypes.objectOf(PropTypes.node),`, will already throw a warning